### PR TITLE
Add unity salt-state

### PR DESCRIPTION
**Pull request by wbrawner**

### CODE REVIEW

File: salt/unity/init.sls [hunk 1]
```diff
@@ -0,0 +1,11 @@
+unity:
+    file.managed:
+        - name: /opt/Unity-{{ grains.version }}.pkg
```
Comment (thusoy) [inline salt/unity/init.sls]: The version and hash should be specified in pillar like we do for other versioned stuff, ie something like
```yaml
{% set version_info = pillar.get('unity', {}).get('version_info', '2017.2 sha256=deadbeef') %}
{% set version, version_hash = version_info.split() %}
```

File: salt/unity/init.sls [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        - source: https://netstorage.unity3d.com/unity/{{ grains.hash }}/MacEditorInstaller/Unity-{{ grains.version }}.pkg
+        - source_hash: {{ grains.source_hash }}
+    cmd.watch:
```
Comment (thusoy) [inline salt/unity/init.sls]: Add a newline before this to keep the statements separated and easier to read.

**Comment by wbrawner** [comment]
@thusoy Added the Android & iOS salts now, ready for another review

**Comment by wbrawner** [comment]
Merged #17

**Pull request merged**